src/lib.rs

```diff
@@ -501,7 +501,7 @@ pub struct FlamegraphOptions {
     pub notes: Option<String>,
 
     /// Omit functions smaller than <FLOAT> pixels
-    #[clap(long, default_value = "0.01f32", value_name = "FLOAT")]
+    #[clap(long, default_value = "0.01", value_name = "FLOAT")]
```

Comment:
This seems unrelated?

Reply:
Yes, it was throwing runtime error for me, probably because "0.01f32" doesn't parse to `f64`? But you are right, it's unrelated. Should I put it into a separate commit?

Reply:
Yes, splitting it out in a separate commit would be fine, thanks!